src/content/foundation/Contact.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Contact the GraphQL Foundation
+sidebarTitle: Contact us
+heroText: The GraphQL Foundation is here to help.
+layout: foundation
+category: GraphQL Foundation
+permalink: /foundation/contact/
+sublinks: Membership Inquriries,Membership Support,Meeting Logistics,Media Inquries,Technical Issues,General
+next: /foundation/join/
+---
+
+The GraphQL Foundation is supported by staff from [The Linux Foundation](https://linuxfoundation.org). If you have questions, please reach out to us using the links below.
+
+*Please note that we do not provide user support for GraphQL. If you have an implementation question,
+please use the [relevant community channels](/community/).*
+
+## Membership Inquiries
+
+If you would like to learn more about becoming a GraphQL member, please email
+[membership@graphql.org](mailto:membership@graphql.org).
+
+## Membership Support
+
+If you are already a member of the [GraphQL Foundation](./) and you need help with your membership, please reach out to [operations@graphql.org](mailto:operations@graphql.org).
```

Comment:
Cannot use relative link here since this page `https://deploy-preview-1019--graphql-org.netlify.app/foundation/contact` auto-redirects to `https://deploy-preview-1019--graphql-org.netlify.app/foundation/contact/`, so `./` links to itself.

```suggestion
If you are already a member of the [GraphQL Foundation](/foundation/) and you need help with your membership, please reach out to [operations@graphql.org](mailto:operations@graphql.org).
```

Reply:
Good catch, that was a copy and paste error.